components/integration/base/delete-snapshots.yaml

```
@@ -0,0 +1,26 @@
+kind: ClusterRole
+apiVersion: authorization.openshift.io/v1
+metadata:
+  name: delete-snapshots
+rules:
+  - apiGroups:
+    - "appstudio.redhat.com"
+    resources:
+      - "snapshots"
+    verbs:
+      - "delete"
+      - "deletecollection"
+---
+kind: RoleBinding
+#apiVersion: authorization.openshift.io/v1
+apiVersion: authorization.openshift.io/v1
+metadata:
+  name: delete-snapshots
+  namespace: integration-service
+subjects:
+  - kind: Group
+    name: integration-team
```

The name of the team is "Integration team"
```suggestion
  - apiGroup: rbac.authorization.k8s.io
    kind: Group
    name: Integration team
```